.babelrc

```diff
@@ -4,9 +4,10 @@
   "comments": false,
   "presets": [
     "react-native",
+    "react-native-stage-0/decorator-support"
   ],
   "plugins": [
-    "./plugins/babelRelayPlugin"
+    ["relay", {"compat": true, "schema": "data/schema.json"}],
```

Comment:
I'm reaching this issue here: https://github.com/facebook/relay/issues/1665

I'll try without `compat` mode